src/firestore/indexes.ts

```diff
@@ -334,11 +339,17 @@ export class FirestoreIndexes {
    * Validate that an arbitrary object is safe to use as an {@link Spec.FieldOverride}.
    * @param field
    */
-  validateField(field: any): void {
+  validateField(field: any, collectionsWithTtl: Set<string>): void {
     validator.assertHas(field, "collectionGroup");
     validator.assertHas(field, "fieldPath");
     validator.assertHas(field, "indexes");
 
+    /** Only one TTL per Collection */
```

Comment:
Is this check needed? Backend has the same check and should delegate this to backend.

Reply:
It was agreed to have it in the design doc. 

The deployment of the indexes & field overrides is sequential. In a first pass all fields are validated. 
Then a second pass will be done for actual patching. I added this check to catch it earlier and show an error message from CLI instead of 4xx, 5xx codes.

Removing the asserts for some edge cases is not a problem. It would prompt a 403 or 500 error with the same messages as GCP console.

Reply:
IMO it's better to let backend decide. This is not an accurate check either since right now we have 1 `UP` ttl config per collection. This limit could also be changed in the future.

Reply:
SG. I'll remove this check. The HTTP error has clear information on this.